games-engines/gemrb/additional-files/cmake_install.cmake

```diff
@@ -0,0 +1,96 @@
+    set(CMAKE_INSTALL_COMPONENT)
+  endif()
+endif()
+
+# Is this installation the result of a crosscompile?
+if(NOT DEFINED CMAKE_CROSSCOMPILING)
+  set(CMAKE_CROSSCOMPILING "FALSE")
+endif()
+
+# Set default install directory permissions.
+if(NOT DEFINED CMAKE_OBJDUMP)
+  set(CMAKE_OBJDUMP "/bin/objdump")
+endif()
+
+if(NOT CMAKE_INSTALL_LOCAL_ONLY)
+  # Include the install script for the subdirectory.
+  include("/sources/gemrb-0.9.3/build/gemrb/cmake_install.cmake")
+endif()
+
+if(CMAKE_INSTALL_COMPONENT STREQUAL "Unspecified" OR NOT CMAKE_INSTALL_COMPONENT)
+  list(APPEND CMAKE_ABSOLUTE_DESTINATION_FILES
+   "/packages/gemrb-0.9.3-1/.self/documentation/man/man6/gemrb.6")
+  if(CMAKE_WARN_ON_ABSOLUTE_INSTALL_DESTINATION)
+    message(WARNING "ABSOLUTE path INSTALL DESTINATION : ${CMAKE_ABSOLUTE_DESTINATION_FILES}")
+  endif()
+  if(CMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION)
+    message(FATAL_ERROR "ABSOLUTE path INSTALL DESTINATION forbidden (by caller): ${CMAKE_ABSOLUTE_DESTINATION_FILES}")
+  endif()
+  file(INSTALL DESTINATION "/packages/gemrb-0.9.3-1/.self/documentation/man/man6" TYPE FILE MESSAGE_NEVER FILES "/sources/gemrb-0.9.3/build/gemrb.6")
+endif()
+
+if(CMAKE_INSTALL_COMPONENT STREQUAL "Unspecified" OR NOT CMAKE_INSTALL_COMPONENT)
+  list(APPEND CMAKE_ABSOLUTE_DESTINATION_FILES
+   "/packages/gemrb-0.9.3-1/.self/documentation/gemrb/README.md;/packages/gemrb-0.9.3-1/.self/documentation/gemrb/INSTALL;/packages/gemrb-0.9.3-1/.self/documentation/gemrb/COPYING;/packages/gemrb-0.9.3-1/.self/documentation/gemrb/NEWS;/packages/gemrb-0.9.3-1/.self/documentation/gemrb/AUTHORS")
+  if(CMAKE_WARN_ON_ABSOLUTE_INSTALL_DESTINATION)
+    message(WARNING "ABSOLUTE path INSTALL DESTINATION : ${CMAKE_ABSOLUTE_DESTINATION_FILES}")
+  endif()
+  if(CMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION)
+    message(FATAL_ERROR "ABSOLUTE path INSTALL DESTINATION forbidden (by caller): ${CMAKE_ABSOLUTE_DESTINATION_FILES}")
+  endif()
+  file(INSTALL DESTINATION "/packages/gemrb-0.9.3-1/.self/documentation/gemrb" TYPE FILE MESSAGE_NEVER FILES
+    "/sources/gemrb-0.9.3/README.md"
+    "/sources/gemrb-0.9.3/INSTALL"
+    "/sources/gemrb-0.9.3/COPYING"
+    "/sources/gemrb-0.9.3/NEWS"
+    "/sources/gemrb-0.9.3/AUTHORS"
+    )
+endif()
+
+if(CMAKE_INSTALL_COMPONENT STREQUAL "Unspecified" OR NOT CMAKE_INSTALL_COMPONENT)
+  list(APPEND CMAKE_ABSOLUTE_DESTINATION_FILES
+   "/packages/gemrb-0.9.3-1/.self/bin/extend2da.py")
+  if(CMAKE_WARN_ON_ABSOLUTE_INSTALL_DESTINATION)
+    message(WARNING "ABSOLUTE path INSTALL DESTINATION : ${CMAKE_ABSOLUTE_DESTINATION_FILES}")
+  endif()
+  if(CMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION)
+    message(FATAL_ERROR "ABSOLUTE path INSTALL DESTINATION forbidden (by caller): ${CMAKE_ABSOLUTE_DESTINATION_FILES}")
+  endif()
+  file(INSTALL DESTINATION "/packages/gemrb-0.9.3-1/.self/bin" TYPE FILE MESSAGE_NEVER PERMISSIONS OWNER_WRITE OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE FILES "extend2da.py")
```

Comment:
Would be nice if the DESTINATION paths would use a variable instead of a hardcoded path